### PR TITLE
#688 [ Fixed ] "Community leaders" Does not have "Dot" Pagination

### LINKED ISSUE #688: "Community leaders" Does not have "Dot" Pagination.
**Describe the bug**

"Community leaders" Only have one "Dot" for pagination. 
**To Reproduce**

1.Go to 'https://developers.italia.it/''
2.Click on 'Platforms'
3.Click on 'Read more' from any items. I clicked "SPID"

**Expected behavior**

"Community leaders" should have two "Dot" for pagination. 
**Screenshots**

image

**Smartphone (please complete the following information):**
Lenovo Thinkpad
Windows 10
Mozilla Firefox
80

**Additional context**
Add any other context about the problem here.-->

### PR DESCRIPTION
**Pull request by UchihaCFC**
## Description

The github member navigation in the sections within "Platforms" now works correctly.

The problem was that the variable `indicators` always arrived as 0 when the team members were 6 or more, and it did not correctly perform the comparison to add more navigation circles in the carrousel, since it began to count from 1 in the following way `{1...indicators}`.

One more module was assigned when the number of team members is greater than 5.

## Checklist

- [x] I followed the indications in the [CONTRIBUTING](../CONTRIBUTING.md)
- [x] The documentation related to the proposed change has been updated accordingly (also comments in code).
- [x] Have you written new tests for your core changes, as applicable?
- [x] Have you successfully ran tests with your changes locally?
- [x] Ready for review! :rocket:

## Fixes
- Fixes #688

### CODE REVIEW

File: _includes/github_team_members.html [hunk 1]
```diff
@@ -35,12 +35,15 @@ <h6 class="mb-3">{{  t.project_leader }} </h6>
 		<div class="carousel-footer mt-auto d-flex">
 
 			<ol class="carousel-indicators align-self-center">
-				{% assign indicators = team_members.size | modulo: 3 %}
-				{% assign checkIndicators = indicators | times: 3 %}
+        {% assign indicators = team_members.size | modulo: 3 %}
```
Comment (bfabio) [inline _includes/github_team_members.html:38]: Hello @UchihaCFC, thanks for the PR!

I'm not sure about this. Can't we just do `{% assign indicators = team_members.size | divided_by: 3.0 | ceil %}` and simplify the whole thing?

Comment (UchihaCFC) [reply _includes/github_team_members.html]: You are right, it is much cleaner! In this type of templates I did not know that there was \`dividev\_by\`, thanks @bfabio, I will be really useful in the future :).

I uploaded one more commit with this change after testing that everything worked ok.

**Review by bfabio — APPROVED** [review]
lgtm :+1:

**Review by sebbalex — APPROVED** [review]

**Comment by bfabio** [comment]
@UchihaCFC Thanks for your work! Merged :rocket:

**Pull request merged**